package.json

```diff
@@ -42,51 +42,33 @@
     "lint:hbs": "ember-template-lint .",
     "lint:js": "eslint .",
     "test": "yarn test-fast && yarn test-slow",
-    "test-fast": "mocha node-tests/unit/**/*.js ./node-tests/integration/**/*.js",
+    "test-fast": "mocha node-tests/integration/**/*.js",
     "test-slow": "mocha ./node-tests/acceptance/**/*.js"
   },
   "dependencies": {
-    "broccoli-file-creator": "^2.1.1",
-    "broccoli-funnel": "^2.0.2",
-    "broccoli-merge-trees": "^3.0.2",
-    "broccoli-source": "^3.0.0",
+    "@electron-forge/cli": "^6.0.0-beta.45",
     "broccoli-string-replace": "^0.1.2",
-    "capture-exit": "^2.0.0",
     "chalk": "^3.0.0",
-    "console-ui": "^2.2.3",
-    "core-object": "^3.1.0",
-    "electron-forge": "~5.2.4",
+    "debug": "^4.1.1",
+    "denodeify": "^1.2.1",
     "electron-protocol-serve": "^1.3.0",
     "ember-cli-babel": "^7.11.1",
-    "ember-cli-htmlbars": "^4.0.5",
-    "ember-cli-version-checker": "^3.1.3",
     "ember-inspector": "^3.12.5",
-    "execa": "^3.3.0",
     "file-url": "^3.0.0",
-    "fs-extra": "^8.1.0",
-    "globby": "^10.0.1",
-    "lodash": "^4.15.0",
-    "quick-temp": "^0.1.5",
+    "ncp": "^2.0.0",
+    "portfinder": "^1.0.25",
     "rimraf": "^3.0.0",
```

Comment:
I think `rimraf` can get moved to `devDependencies` as well, right?

Reply:
No, we use it in the electron-forge template which runs from the blueprint.